gnwmanager/cli/_sdcard.py

```diff
@@ -0,0 +1,31 @@
+import logging
+from pathlib import Path
+
+from cyclopts import Parameter, validators
+from typing_extensions import Annotated
+
+from gnwmanager.cli._parsers import GnWType
+from gnwmanager.cli.main import app
+
+log = logging.getLogger(__name__)
+
+
+@app.command(group="SD Card (Only for game and watch with SD Card mod)")
+def sdpush(
+    file: Annotated[Path, Parameter(validator=validators.Path(exists=True, dir_okay=False))],
```

Comment:
Do we want this to command with multiple files? E.g.

```
$ gnwmanager sdpush build/* /path/on/device
```

I suppose we can do this in a future PR; I have some upcoming updates for cyclopts v3 that makes this easier.

Reply:
It would be a very useful feature for sure ! Let's do that after this PR ;)